### PR TITLE
docs: remove references to drawing_tool in Text class

### PR DESCRIPTION
**Pull request by ReblochonMasque**
reverted all Nov 2 commits save part of  `557196f` examples/vehicle_dim.py, delete  `# drawing_tool.set_grid(True)`

installed and tested `pre-commit`, which issued a warning:

> [WARNING] The 'rev' field of repo 'https://gitlab.com/pycqa/flake8' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.

running `pre-commit autoupdate` as suggested removed the warning, but modified some files.  I did not want to commit them, but the following error gave me no choice:

> [ERROR] Your pre-commit configuration is unstaged.

I believe these changes were okay so I committed them.

In short, this PR changes the `class Text` docstring in `_text.py`, and deleted `# drawing_tool.set_grid(True)` in `examples/beam1.py`

I merged master into this branch (3 commits) so it can be merged directly, if so you choose.

### CODE REVIEW

**Comment by rvodden** [comment]
Great! Thanks for the contribution. Your commit history is pretty untidy. Do you mind if sort that out for you? We don't need all those "precommit test" commits in our history, for example. Similarly, we use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) so that release notes can be automatically generated.

**Comment by ReblochonMasque** [comment]
Yes, please - the reason for this is that I was setting up the pre-commit machinery when I did not have additional commits to make, and I wanted to make sure it worked properly prior to PR.
Then, IDK how to tidy the mess; sorry!

**Comment by rvodden** [comment]
@ReblochonMasque for reference, I'll show you what I did to fix this.

This is the commit tree for your purge_examples branch (I've cloned your fork locally):

Screenshot 2021-11-04 at 06 27 25

As you can see it's pretty messy with various merges (its always better to rebase :-) ) intertwined with real changes. Firstly I checked out the commit on which you first based the branch, and created a new branch from that point:

```
git checkout 7bf839b
git checkout -b repair
```

Now I can copy the commits we want on to that new branch. Looking through the list there is one we want, `ab476aa` and ideally we'd have the commit with the drawing_tool comments removed too, but this isn't in an isolated commit, so we can just mop that up at the end. To grab the commit we do want:

```
git cherry-pick ab476aa
```

Then tweak the commit message to comply with our conventions:

```
git commit --amend -m "docs: remove references to drawing_tool from Text class"
```

Now I have a nice clean branch. So I update purge_examples to point to this commit pointer:

```
git checkout purge_examples
git reset --hard repair
```

Lastly, this branch is now based on place which master was a while ago not the latest version of master from my fork. Your fork has commits in master which are not in mine, so I add my repo as a new remote called `upstream`

```
git remote add upstream git@github.com:rvodden/pysketcher.git
```

And then reset master so that it points to the same place as my repo:

```
git checkout master
git pull upstream master
```

Switch back to our new purge_examples branch and rebase it of the new master:

```
git checkout purge_examples
git rebase master
```

RIght now all these changes are just local, we've published none of our changes. So lastly lets force push onto your remote which will update the PR (we have to force push as we're editing history):

```
git push origin purge_examples -f
```

And now the remote branch (and the PR) only contain that commit :-)

**Comment by rvodden** [comment]
LGTM - thank you for the contribution :-)

**Comment by ReblochonMasque** [comment]
Great, thank you for spending the time to teach me - I am afraid this is still above my pay grade, but I'll try to learn.

You've rephrased the commits' comments so they "match requirements/standards", but I could not find the requirements. Can you point them to me so I can save you the trouble in the future?

Could you please see if you could provide an answer to [this discussion](https://github.com/rvodden/pysketcher/discussions/318) opened Nov 1st? Maybe some bullet point guidelines if you don't have much time? that would be quite useful to help me evaluate my involvement.

Many thanks
